packages/grafana-data/src/types/fieldOverrides.ts

```diff
@@ -78,6 +78,8 @@ export interface FieldConfigEditorProps<TValue, TSettings extends {}>
   item: FieldConfigPropertyItem<any, TValue, TSettings>; // The property info
   value: TValue;
   context: FieldOverrideContext;
+  min?: number;
+  max?: number;
```

Comment:
This seems wrong to put on directly on `FieldConfigEditorProps`. You should only allow it on numeric fields (props for `NumberValueEditor`).